azure-pipelines.yml

```diff
@@ -47,4 +47,5 @@ stages:
         tags: |
           $(BRANCH_NAME)
           $(BRANCH_NAME)_$(buildTimestamp)_b$(Build.BuildId)
+          latest
```

Comment:
Ok, why do we need latest? Why don't we use master of this repo?

Reply:
removed it